wallet/tickets.go

```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/decred/dcrwallet/walletdb"
 )
 
+// TicketVoteBitsUpdate is a struct containing updates to ticket vote bits
+// settings in the stake database.
+type TicketVoteBitsUpdate struct {
```

Comment:
I think it would make more sense to create a new type that contains both the regular and extended vote bits.